### PR TITLE
UI Enhancements

### LINKED ISSUE #37: Add shadows to UIViews in storyboards
[reference](https://stackoverflow.com/questions/42244040/swift-3-xcode-storyboard-add-box-shadow-to-uiview-like-a-css-style-box-shadow)

tldr:
``` Swift
YourView.layer.shadowOpacity = 0.7
YourView.layer.shadowOffset = CGSize(width: 3, height: 3)
YourView.layer.shadowRadius = 15.0
YourView.layer.shadowColor = UIColor.darkGray.cgColor
```

### PR DESCRIPTION
**Pull request by yaosarayin**
Added Shadow Views, Closes #37 
More commits to come

### CODE REVIEW

File: rideshare/Controllers/RideOfferingsViewController.swift [hunk 1]
```diff
@@ -12,14 +12,26 @@ class RideOfferingsViewController: UIViewController, UITableViewDelegate, UITabl
     
     var rides = [PFObject]()
     var selectedPost: PFObject!
-
+    
+    @IBOutlet weak var FilterButton: UIButton!
```
Comment (nashirj) [inline rideshare/Controllers/RideOfferingsViewController.swift:16]: these should probably be named `filterButton` and `sortButton` so it's clear that they're not a class or something! that said I know we discussed briefly removing the ability to filter/sort for the demo. do we want to keep the buttons in for this saturday's demo day? would appreciate your opinion as well @tahaafzal5

File: rideshare/Extensions/ShadowView.swift [hunk 2]
```diff
@@ -0,0 +1,23 @@
+//
```
Comment (nashirj) [inline rideshare/Extensions/ShadowView.swift:1]: This file should be named `UIView+addShadow.swift` to follow the convention we have for our other extensions!

**Review by nashirj — CHANGES_REQUESTED** [review]

**Comment by nashirj** [comment]
Yao this looks so good!! Thank you for working on this. A couple request in addition to my comments above if you have some time to work more on these things:
1) When I scroll the table view all the way to the top or bottom, there is white space showing behind the table view. Can you maybe set the background color of the screen to the same color as the table view, or something so that this doesn't happen?
top-tableview
bottom-tableview
2) Can you figure out a way to make the destination/pickup labels not overlap as is shown in this picture?
overlapping-text
3) Can you do all this stuff on the account details and profile page as well for consistency's sake?
account-details-and-profile

Thank you so much for your fixes, it all looks SO good!!

File: rideshare/Controllers/RideOfferingsViewController.swift (hunk 1 above)
Comment (tahaafzal5) [reply rideshare/Controllers/RideOfferingsViewController.swift:16]: We can keep them (to show as a future feature in the demo) or get rid of them - I don't mind anyway we choose to go.

**Comment by yaosarayin** [comment]
To Do:
- Make card view cells for my profile page
- Redesign ride details page incorporating the map
- Figure out a way to display long location names

**Comment by nashirj** [comment]
Also please fix the date picker

**Comment by nashirj** [comment]
Consistent color scheme for all pages that will be shown in the demo

**Pull request merged**